### DISCUSSION TITLE
Search didnt work (Search failed. Please try again later.)

### DISCUSSION THREAD

**Discussion by slither-tuft** [community/community#58623] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I am a security researcher. I was doing research for a new whitepaper I am going to write. GitHub is no longer responding to any of my searches. when I try from another account, there is no problem. I run my queries by opening many tabs at the same time. could there be a ban etc., for this reason?

Please help me as soon as possible. why am I having such a problem?

![image](https://github.com/community/community/assets/135229645/7d1a2eb2-49fc-4881-af24-6fef24051355)

**Comment by look** [comment]
Which account are you having problems from?

**Reply by slither-tuft** [reply]
slither-tuft

**Comment by maqsats** [comment]
I am also having this issue from my current account

**Comment by yethranayeh** [comment]
Also facing this issue. I get this error for any code search I try, it returns 0 results and this error. However, all my searches show results for Repositories, Issues, Discussions, etc. Only no results for Code.

**Reply by look** [reply, marked as answer]
There is an [ongoing incident that is affecting code search](https://www.githubstatus.com/incidents/myvz2tsj2dh8).

**Comment by Supmaxred** [comment]
same

**Comment by LiteBrite82** [comment]
Hi @maqsats, @yethranayeh, and @Supmaxred 

Thanks for the reports! 
I created a pinned post to catch all comments related to the current disruption. 
[GitHub Code Search is experiencing a disruption - 10/11/2024 #141212](https://github.com/orgs/community/discussions/141212)
